code/modules/projectiles/projectile/rocket.dm

```diff
@@ -26,7 +27,111 @@
 		sleep(picked_up_speed)
 
 /obj/item/projectile/rocket/to_bump(var/atom/A)
-	explosion(A, 1, 3, 5, 8) //RPGs pack a serious punch and will cause massive structural damage in your average room, but won't punch through reinforced walls
+	if(explosive == 1)
+		explosion(A, 1, 3, 5, 8) //RPGs pack a serious punch and will cause massive structural damage in your average room, but won't punch through reinforced walls
+		if(!gcDestroyed)
+			qdel(src)
+	else
+		..()
+		if(!gcDestroyed)
+			qdel(src)
+
+/obj/item/projectile/rocket/lowyield
+	name = "low yield rocket"
+	icon_state = "rpground"
+	explosive = 0
+	damage = 30
+	stun = 5
+	weaken = 5
+	damage_type = BRUTE
+	nodamage = 0
+	flag = "bullet"
+	fire_sound = 'sound/weapons/rocket.ogg'
```

Comment:
defining `fire_sound` is also unnecessary, don't have to re-define any of the vars that are the same as the parent class.